zulipterminal/model.py

```diff
@@ -339,6 +342,9 @@ def set_search_narrow(self, search_query: str) -> None:
         self.unset_search_narrow()
         self.narrow.append(["search", search_query])
 
+    def get_alert_words(self) -> List[str]:
+        return self._alert_words
```

Comment:
We need to return a copy here, to ensure the data isn't modified by a caller of this function.